qgis_fmask_sentinel2Stacked.py

```diff
@@ -73,4 +82,8 @@
         pass
 
 progress.setConsoleInfo('Done.')
-dataobjects.load(output, os.path.basename(output))
+dataobjects.load(outpath, os.path.basename(outpath))
+
+
+#OutputFile|output|Output cloud mask|tif
```

Comment:
Don't leave commented-out code. With version control you do not need to do that to keep old code around.